.chronus/changes/feature-source-file-header-2025-2-27-1-49-36.md

```diff
@@ -0,0 +1,10 @@
+---
+changeKind: feature
+packages:
+  - "@alloy-js/core"
+  - "@alloy-js/typescript"
+---
+
+- Support SourceFile header. This PR adds the following components:
```

Comment:
In future lets add separate changelog files for each change, this will mangle the changelog entries (Prose is not added to core for example).